mingw-w64-git/PKGBUILD

```diff
@@ -97,18 +121,20 @@ i686)
 x86_64)
   SHAREDIR=/mingw64/share/git
   ;;
+aarch64)
+  SHAREDIR=/clangarm64/share/git
+  ;;
```

Comment:
I wonder whether we can just replace this entire `case` block with `SHAREDIR=/$MINGW_ARCH/share/git`. What do you think?